src/base.ts

```diff
@@ -74,12 +76,14 @@ export interface ApiCallOptions extends Omit<RequestInit, "method" | "body"> {
 
 interface AuthorizedApiCallOptions extends ApiCallOptions {
   isAuthorizedRequest?: boolean
+  apiVersion?: ApiVersion
 }
 
 type Method = "GET" | "POST" | "PUT" | "PATCH" | "DELETE"
 
 interface RequestOptions extends AuthorizedApiCallOptions {
   method?: Method
+  apiVersion?: ApiVersion
```

Comment:
Do we need to add this twice? It should already be available here since you're extending `AuthorizedApiCallOptions`